src/test/java/io/javalin/metrics/JavalinMetricsTest.kt

```diff
@@ -0,0 +1,15 @@
+package io.javalin.metrics
+
+import io.javalin.Javalin
+import io.javalin.util.TestUtil
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class JavalinMetricsTest {
+
+    @Test
+    fun `enable javalin metrics`() = TestUtil.test(Javalin.create().enableMetrics()) { app, http ->
```

Comment:
@tipsy this is very basic. Should we test with an actual registry implementation like `io.micrometer:micrometer-registry-prometheus:1.1.3`?
But that would add much overhead, I'm not sure about this.. :thinking:

Reply:
nvm maybe I can test with an in memory registry. I will check tomorrow

Reply:
I don't think it's required.